app/riot/src/client/preview/render-riot.js

```diff
@@ -0,0 +1,72 @@
+import { document } from 'global';
+
+export function guessRootName(stringified) {
+  const whiteSpaceLocation = stringified.indexOf(' ', stringified.indexOf('<') + 1);
+  const firstWhitespace = whiteSpaceLocation === -1 ? stringified.length : whiteSpaceLocation;
+  const supposedName = stringified.trim().match(/^<[^ >]+\/>$/)
+    ? stringified.trim().replace(/[<>/]/g, '')
+    : stringified.substring(
+        stringified.indexOf('<') + 1,
+        Math.min(firstWhitespace, stringified.indexOf('>'))
+      );
+  const matchingBuiltInTag = document.createElement(supposedName).constructor.name;
+  return matchingBuiltInTag === 'HTMLUnknownElement' ? supposedName : 'root';
+}
+
+export function renderStringified(
+  { tags, scenario = `<${(tags[0] || []).boundAs || guessRootName(tags[0] || '')}/>` },
```

Comment:
Is `scenario` a known thingy among `riot` users, or we can call it `template` like in `vue` and `angular` ?

Reply:
yes, ideally